js/everything.js

```diff
@@ -402,6 +410,28 @@ function loadTime() {
     }
 }
 
+// Key to the persisted volume level entry in localStorage
+var LOCAL_STORAGE_VOLUME_KEY = "VOLUME_LEVEL";
+
+// The delay to debounce writes of the volume level to localStorage
+var VOLUME_PERSIST_DELAY = 200;
+
+// The timeout ID tracking debounced writes of volume level to localStorage
+var debouncedPersistVolumeTimer;
+
+function storeVolume(volume) {
+    if (debouncedPersistVolumeTimer) {
```

Comment:
How important do you think it is to limit writing to `localStorage` by time? I'm not very familiar with the API

In my head I was thinking of doing something simpler, (pseudocode):

```javascript
if (currentVolume !== localStorage.getItem("volume")) {
    localStorage.setItem("volume", currentVolume);
}
```